### PR TITLE
[TACHYON-1431] Fix NPE in BlockInfo tab

### PR DESCRIPTION
**Pull request by aaudiber**
https://tachyon.atlassian.net/browse/TACHYON-1431

"orderedTierAliases" needs to be set even when the offset and limit aren't specified.

### CODE REVIEW

File: servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java [hunk 1]
```diff
@@ -122,8 +125,6 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         uiFileInfos.add(getUiFileInfo(tFS, fileId));
       }
       request.setAttribute("fileInfos", uiFileInfos);
```
Comment (yupeng9) [inline servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java]: Shall this be moved outside, too?

Comment (calvinjia) [inline servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java]: I think we check for `null` in the jsp, we should probably define a contract of which params are guaranteed to be set and which are optional

**Comment by calvinjia** [comment]
LGTM

File: servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java (hunk 1 above)
Comment (yupeng9) [inline servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java]: SGTM.

**Comment by yupeng9** [comment]
LGTM

**Comment by aaudiber** [comment]
Jenkins, test this please.

**Pull request merged**